README.md

```diff
@@ -1,6 +1,6 @@
-# Awesome DevC Contributors
+## Awesome DevC Contributors
```

Comment:
Hey let the single '#' remain na?
As that is the title of the repo not a subtitle!

Reply:
its just a quick small change so that I can merge it directly!